### PR TITLE
At 3 homepage

### PR DESCRIPTION
**Pull request by Clombardi91**
Map appears at top of page and centers on the bull statue in Durham.

### CODE REVIEW

File: places/views.py [hunk 1]
```diff
@@ -1,3 +1,20 @@
 from django.shortcuts import render
+from django.http import JsonResponse
```
Comment (KimWheaton) [inline places/views.py]: Can consolidate imports from same module

File: static/css/home.css [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Quicksand", sans-serif;
+    font-size: 62.5%;
+    font-size: 10px;
+}
+/*-- Inspiration taken from abdo steif -->
+/* --> https://codepen.io/abdosteif/pen/bRoyMb?editors=1100*/
+
+/* Navbar section */
```
Comment (KimWheaton) [inline static/css/home.css]: Is the nav code still needed?

File: static/css/opening.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/* html {
```
Comment (KimWheaton) [inline static/css/opening.css]: This file can be deleted from this branch. A backlog ticket was added and this feature will be worked on later.

File: static/js/opening.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+// let splitText = new SplitText('h1', {type: 'chars'});
```
Comment (KimWheaton) [inline static/js/opening.js]: This file can be deleted from this branch. This is part of the same backlog ticket I mentioned above.

File: templates/places/hamburger.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{% extends 'base.html' %}
```
Comment (KimWheaton) [inline templates/places/hamburger.html]: It looks like the hamburger script was added to the homepage and you don't need this file.

File: templates/places/opening.html [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<!-- {% extends 'base.html' %}
```
Comment (KimWheaton) [inline templates/places/opening.html]: See my comment above about this feature.

**Review by KimWheaton — CHANGES_REQUESTED** [review]
Great job, Chris! The map looks great. It looks like there is code that isn't being used and can be removed. Please see my comments.

File: static/css/home.css (hunk 2 above)
Comment (Clombardi91) [reply static/css/home.css]: I deleted it

File: static/css/opening.css (hunk 3 above)
Comment (Clombardi91) [reply static/css/opening.css]: This one is deleted.

File: static/js/opening.js (hunk 4 above)
Comment (Clombardi91) [reply static/js/opening.js]: This file is deleted.

File: templates/places/hamburger.html (hunk 5 above)
Comment (Clombardi91) [reply templates/places/hamburger.html]: The script has been removed.

File: templates/places/opening.html (hunk 6 above)
Comment (Clombardi91) [reply templates/places/opening.html]: This has been taken care of.

**Review by allisondurham — APPROVED** [review]

**Pull request merged**